lua/lspconfig/configs/lsp_ai.lua

```diff
@@ -0,0 +1,27 @@
+return {
+  default_config = {
+    cmd = { 'lsp-ai' },
+    filetypes = {},
+    root_dir = nil,
+    single_file_support = true,
+    init_options = {
+      memory = {
+        file_store = vim.empty_dict(),
+      },
+      models = vim.empty_dict(),
+    },
+  },
+  docs = {
+    [[
+https://github.com/SilasMarvin/lsp-ai
+
+LSP-AI is an open source language server that serves as a backend for AI-powered functionality in your favorite code
+editors. It offers features like in-editor chatting with LLMs and code completions.
```

Comment:
How does chat work? Is that a custom lsp request?

Reply:
There's one available (`document/textGeneration` IIRC) but it feels somewhat low-level, The more user friendly way is to define a prefix in config that enables the chat code action when present in the buffer.